A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -185,13 +185,35 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
 
         this.A32NXCore.update();
 
+        this.updateMCDU();
+
         this.updateAutopilot();
 
         this.updateScreenState();
 
         this.updateGPSMessage();
     }
 
+    // check if init2 page is active
+    updateMCDU() {
+        if (!this.initB) {
+            if (this.isLockedInit()) {
+                if (!this.initBTimer) {
+                    this.initBTimer = 0;
+                }
+                if (this.initBTimer > 15) {
+                    if (this.getTitle() === "INIT FUEL PREDICTION {}") {
```

Comment:
This is not a reselient way of checking if the INIT B page is showing. The title changes based on certain conditions. A better way is to check which page is showing on the mcdu. I will add code later when I got time

Reply:
used set variable instead.